src/wallet/wallet.cpp

```diff
@@ -3063,6 +3048,30 @@ bool CWallet::CreateTransaction(const std::vector<CRecipient>& vecSend, CWalletT
     return true;
 }
 
+bool CWallet::CreateTransaction(const std::vector<CRecipient>& vecSend, CWalletTx& wtxNew, CReserveKey& reservekey, CAmount& nFeeRet,
```

Comment:
I think this is avoidable complexitiy, because `avoidpartial` will be always higher fees.
I think it is OK, you pay fee for privacy, while giving incentive to privacy over fee by default.

Reply:
Actually `avoidpartial` gives the same fee a lot of the times, from what I've seen so far. It depends on your utxo set, of course.